### ISSUE TITLE
[BUG] netlab initial crashes when 'modprobe' command is not available

### ISSUE THREAD

**Issue by ipspace** [ipspace/netlab#1219]
The initial configuration tasklist for FRR invokes **modprobe** to load the VRF module. If that command fails, Ansible reports an error, rescues the task block, but also reports an error status which crashes 'netlab initial'.

<img width="637" alt="image" src="https://github.com/ipspace/netlab/assets/16917911/d659a198-7422-4cc0-9126-bddecf28ed6e">

<img width="1036" alt="image" src="https://github.com/ipspace/netlab/assets/16917911/9aaaaf78-d01e-4cde-8b43-0e38f1fd889e">

Sample topology:

```
provider: clab
nodes:
  rtr:
    device: frr
```

**Comment by ipspace** [comment]
Correction: the minimum lab topology requires two devices. One of them has to be FRR, one of them has to be something else, so that we have a host with a failed task, and another host that's OK:

```
provider: clab
name: test

nodes:
  r1:
    device: frr
  r2:
    device: cumulus

links: [ r1-r2 ]
```

Ansible is as bizzare as it gets...

**Comment by ipspace** [comment]
Fixed in 3287217a7d6d4d0420d609257b5507ccd4310bae
